https://bugzilla.mozilla.org/show_bug.cgi?id=1695863

Should be investigated why this happens on Solaris only.
It's needed also for Thunderbird.

--- firefox-102.1.0/toolkit/components/terminator/nsTerminator.cpp
+++ firefox-102.1.0/toolkit/components/terminator/nsTerminator.cpp
@@ -62,7 +62,7 @@ // Additional number of milliseconds to
 // forcefully.
 #define ADDITIONAL_WAIT_BEFORE_CRASH_MS 3000
 
-#define HEARTBEAT_INTERVAL_MS 100
+#define HEARTBEAT_INTERVAL_MS 1000
 
 namespace mozilla {
 
